### PR TITLE
Nutmeg Upgrade - Code Merge 6/6

### PR DESCRIPTION
**Pull request by shadinaif**
## Change description

Nutmeg Upgrade - Code Merge 6/6

## Type of change
- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)

## Related issues

Related to: https://appsembler.atlassian.net/browse/RED-3373

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: cms/djangoapps/appsembler/tests/test_appsembler_sites_tasks.py [hunk 1]
```diff
@@ -54,23 +54,14 @@ def setUp(self):
...
         self.organization = OrganizationFactory.create(short_name=self.organization_name)
 
-    def get_course_id(self, use_new_format=False):
+    def get_course_id(self):
         """
-        Get the textual course ID in old `Org/Course/Run` format.
-
-        :param use_new_format: Use the new `course-v1:Org+Course+Run` format.
-
-        # TODO: (Nutmeg??) Fix this and ONLY use new course format once Open edX test modulestore fix it
```
Comment (shadinaif) [inline cms/djangoapps/appsembler/tests/test_appsembler_sites_tasks.py:63]: Thanks for the note @OmarIthawi !

File: lms/djangoapps/static_template_view/urls.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
     path('500', views.render_500, name='static_template_view.views.render_500'),
 
     path('blog', views.render, {'template': 'blog.html'}, name="blog"),
-    url(r'^tracks$', views.render, {'template': 'tracks.html'}, name="tracks"),
+    path('tracks', views.render, {'template': 'tracks.html'}, name="tracks"),
```
Comment (OmarIthawi) [inline lms/djangoapps/static_template_view/urls.py:20]: Same here.

File: lms/urls.py [hunk 3]
```diff
@@ -987,22 +987,22 @@
...
 urlpatterns += (
-    url(r'^appsembler/api/', include('openedx.core.djangoapps.appsembler.sites.urls')),
+    path('appsembler/api/', include('openedx.core.djangoapps.appsembler.sites.urls')),
 )
 
 # Tahoe API for registrations, enrollments, and users
 urlpatterns += (
-    url(r'^tahoe/api/',
+    path('tahoe/api/',
         include(('openedx.core.djangoapps.appsembler.api.urls', 'tahoe-api'),
```
Comment (OmarIthawi) [inline lms/urls.py:1006]: needs to be added via the `openedx.core..appsembler.settings` django plugin app

**Review by OmarIthawi — APPROVED** [review]
Looks good to me. Thanks Shadi for the tremendous work here!!

**Pull request merged**